src/publish/confluence/confluence-helper.ts

```diff
@@ -33,6 +34,10 @@ export const FILE_FINDER: RegExp = /(?<=href=\')(.*)(?=\.qmd)/;
 const IMAGE_FINDER: RegExp =
   /(?<=ri:attachment ri:filename=["\'])[^"\']+?\.(?:jpe?g|png|gif|m4a|mp3|txt)(?=["\'])/g;
 
+const capitalizeFirstLetter = (value: string): string => {
+  return value[0].toUpperCase() + value.slice(1);
```

Comment:
Do you need protection against empty strings anywhere in the stack (not sure if this already exists when called, so just adding question as sanity check)

Reply:
Fixed.  Thank you.